app/forms/register/establishers/individual/AddressResultsFormProvider.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.register.establishers.individual
+
+import javax.inject.Inject
+
+import forms.mappings.Mappings
+import models.addresslookup.{Address, AddressRecord}
+import play.api.data.{Form, Forms}
+
+class AddressResultsFormProvider @Inject() extends Mappings {
+
+  def apply(): Form[Address] =
+    Form(
+      "value" -> addressMapping()
```

Comment:
If we use an ID instead of a whole address we just need to make sure that the ID is within the bounds of the set of addresses returned by the address finder API